minicluster/pom.xml

```diff
@@ -123,6 +123,12 @@
       <artifactId>curator-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.curator</groupId>
+      <artifactId>curator-test</artifactId>
+      <version>5.1.0</version>
```

Comment:
versions should be managed in the pom.xml at the root in the dependencyManagement section